contributing.md

```diff
@@ -0,0 +1,136 @@
+# Contributing
+
+> You want to contribute to the website? Welcome!
+> Our website runs off of VuePress. Here you will find contribution resources and advice.
+
+If you have any questions, you can ask them in #website-dev channel in our Discord.
+
+ * [Preparing the development environment](#preparing-the-development-environment)
+ * [Guidelines](#guidelines-1)
+ * [Localization](#localization)
+
+## Preparing the development environment
+### Fork the project
+* Create your own fork of the repository by choosing the **Fork** option at the top right of the website github repository.
+* Create a new branch before working on your new fork. The method of branching depends on the Git CLI/GUI that you use, reference your software manual, or contact us if you need help with this step.
+* Once on your own branch, you're ready to start making improvements to the website
+
+> You can use a git client like [GitHub Desktop](https://desktop.github.com/) to manage your local repository.
```

Comment:
Maybe more personal but I'm not a big fan of GD. I would suggest also recommending the two most popular visual Git clients as I think they're more beginner-friendly and have a nicer interface overall. GitKraken is the only one of these 3 that works natively on Linux too!

```suggestion
> You can use a git client like [GitHub Desktop](https://desktop.github.com/), [GitKraken](https://www.gitkraken.com/) or [Sourcetree](https://www.sourcetreeapp.com/) to manage your local repository.
```

Reply:
I agree with you and use [Git fork](https://git-fork.com/). The question is are these tools really more beginner friendly? 
I believe GD could not be simpler, it only has a fetch, push and commit buttons. On the other hand Git Kraken seems powerful and great but complex.

Linux support is also a good argument.
If you think it would be great to list a few choice for the reader and suggest other tools, I will be happy to add your suggestion.